src/rpc/server.cpp

```diff
@@ -18,11 +18,15 @@
 
 #include <algorithm>
 #include <atomic>
+#include <cassert>
```

Comment:
18452: missing changes in `src/node/interfaces.cpp` (`src/interfaces/node.cpp` in bitcoin)